### PR TITLE
Update to support 0.9.5.0

### PR DESCRIPTION
**Pull request by Utomnia**
Updated to support the Eco 0.9.5.0 playtest

(the supplied unity3d file didn't work for me, so can provide another if others have the same problem)

### CODE REVIEW

File: Whetstone/Whetstone.cs [hunk 1]
```diff
@@ -37,32 +37,32 @@ public abstract class Whetstone : Item
...
-        public override void OnUsed(Player player, ItemStack itemStack)
+        public override string OnUsed(Player player, ItemStack itemStack)
         {
-            base.OnUsed(player, itemStack);
             var selectedItem = player.User.Inventory.Toolbar.SelectedItem;
             if (selectedItem == null)
             {
                 player.Error(Localizer.DoStr($"Missing repair target. Please select a tool on the toolbar"));
-                return;
+                return "";
```
Comment (tony84727) [inline Whetstone/Whetstone.cs:46]: Thanks for the PR.
Just a little bit of nitpicking: maybe use `string.Empty` here is more idiomatic.

**Review by tony84727 — COMMENTED** [review]
Thanks for the PR.
LGTM, just a little bit of nitpicking.

File: Whetstone/Whetstone.cs (hunk 1 above)
Comment (Utomnia) [reply Whetstone/Whetstone.cs:46]: sure, do you want me to submit a new PR?

Comment (tony84727) [reply Whetstone/Whetstone.cs:46]: You can just add a new commit to your branch, or amend the commit to modify. Not necessarily to create a new PR.

Comment (Utomnia) [reply Whetstone/Whetstone.cs]: ok, all done.

**Review by tony84727 — APPROVED** [review]

**Pull request merged**